Common/Sources/Extensions/HyperwalletTransferMethodType.swift

```diff
@@ -49,7 +49,7 @@ public extension HyperwalletTransferMethodType {
                                          "add_transfer_method_processing_time_label".localized())
 
             attributedText.append(value: processingTimeLabel, font: Theme.Label.captionOne, color: color)
-            attributedText.append(value: processingTimeValue, font: Theme.Label.captionOneMedium, color: color)
+            attributedText.append(value: processingTimeValue, font: UIFont.preferredFont(forTextStyle: .caption1).semibold, color: color)
```

Comment:
@whersbach-hw confirmed that since we are facing issues with large text accessibility for captionOneMedium we can use captionOne instead everywhere. So remove captionOneMedium from everywhere and set captionOne. Also pass the font and color to this method ``` func formatFeesProcessingTime(font: UIFont, color: UIColor) -> NSAttributedString ``` and use passed font and color. Also delete SelectTransferMethodTypeConfiguration and modify ``` func configure(configuration: SelectTransferMethodTypeConfiguration?) ``` to ``` func configure(configuration: HyperwalletTransferMethodType?) ``` and delete ``` func getCellConfiguration(indexPath: IndexPath) -> SelectTransferMethodTypeConfiguration ``` from the SelectTransferMethodTypePresenter

Reply:
Changes done